.github/workflows/dart_ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test_consume_*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [ stable, dev ]
```

Comment:
I had to drop 2.7.2 from CI because latest Chrome requires the latest webdriver package which isn't compatible with Dart 2.7, but webdriver is only a dev dependency so this package should continue to work where it currently does.